ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html

```diff
@@ -2,8 +2,9 @@
 <div class="subscribe">
     <div class="container">
         <div class="highlight-box">
+            {% block contents %}
             <h2>{{ _('Subscribe to updates') }}</h2>
-            <div class="subtext">{{ _('Create an account to receive notifications by email or SMS whenever new documents are posted.') }}</div>
+            <div class="subtext">{% block subtext %}{{ _('Create an account to receive notifications by email or SMS whenever new documents are posted.') }}{% endblock %}</div>
```

Comment:
Not sure why you've made these changes? The plan was just to remove the words "or SMS", right?

Reply:
@ChasNelson1990 To extend to subcribe_to_updates.html I have added the blocks. Should I just copy/paste the original and change the text?